packages/carbon-graphs/dev-site/assets/polyfills.js

```diff
@@ -1,16 +1,13 @@
 'use strict';
 
-/* eslint-disable no-plusplus */
-/* eslint-disable prefer-rest-params */
-/* eslint-disable no-restricted-syntax */
-
 (function () {
   // object conversion errors
   const CANNOT_CNVT_TO_OBJ = 'Cannot convert undefined or null to object';
 
   // Shorthand for object.defineProperty
   const objectDefineProperty = function (proto, keyValuePairArray) {
-    for (let i = keyValuePairArray.length; i--;) {
+    // eslint-disable-next-line no-cond-assign
```

Comment:
well it is not necessary to add a condition in here, this for loop**(for (let i = keyValuePairArray.length; i--;)** without condition executes till zero. Since we do not need a condition here I disabled the no condition assign rule.